content/guides/developer-guides/delfin/delfin-kubernetes-guide/_index.md

```diff
@@ -91,18 +91,35 @@ Server Version: v1.21.1 <br />
 
 ---
 
-### Build the delfin docker image locally
+### Get all kubernetes object files required for delfin deployment:
+```bash
+git clone https://github.com/sodafoundation/examples.git
+# examples/delfin-kubernetes/deploy directory contains all the object files of delfin k8s delpoyment
+```
+{{% notice note %}}
+**The default version of delfin used is v1.5.0**
+**To use a specific version of delfin replace the tags of delfin image in `delfin-api-deployment.yaml` ,`delfin-task-deployment.yaml`, `delfin-exporter-deployment.yaml` and `delfin-alert-deployment.yaml` files to desired tag(optional)** <br />
+
+```bash
+# replace tag with required tag in each of the above mentioned files
+image: sodafoundation/delfin:<tag required>
+```
+{{% /notice %}}
+
+
+#### If the user wants to build image from source code manually(optional)
```

Comment:
nit: Please rephrase to  `Build image from source code manually (optional)`

Reply:
changed